### DISCUSSION TITLE
Fake click show on my Short URL

### DISCUSSION THREAD

**Discussion by kashifsaeed79** [YOURLS/YOURLS#3771] [Q&A]
I deploy YOURL. But I face a problem. My YOURL Count the fake Click before Sharing. How I Protect MY YOURL? Even I change my password and  API Key but I still face Fake URL click

**Comment by dgw** [comment]
I don't quite understand what a "Fake URL click" is. However, you should be able to skip logging of "clicks" from any kind of bot that you want by using https://github.com/luixxiul/dont-log-crawlers or making your own fork with a custom list of user-agent strings to ignore. The code also provides for defining CIDR ranges and ASNs to ignore, regardless of UA string.

_Note: The above plugin is marked as archived by its author; I have not tested it. Regardless, it should be easier than writing a new plugin from scratch, even if some compatibility issues with newer YOURLS and/or PHP versions have come up since its author ceased maintenance._
